ways_of_working.md

```diff
@@ -54,21 +51,22 @@ All members are welcome to join these calls! ✨
 
 ### Working Groups and Meetings
 
-- ✅ **Translation and Localisation Meetings**: Every second Wednesday at 14:30 UK time, members of the Translation and Localisation Working Groups share this coworking space to carry out translation work across several languages, and support each other ([Translation Chapter](https://book.the-turing-way.org/community-handbook/translation)).
+- ✅ **Translation and Localisation Meetings**: Every second Tuesday at 16:00 UK time, members of the Translation and Localisation Working Groups share this coworking space to carry out translation work across several languages, and support each other ([Translation Chapter](https://book.the-turing-way.org/community-handbook/translation)).
 - ✅ **Infrastructure Monthly Meetings**: Every second Tuesday of the month, at 16:00 UK time, members of the Infrastructure Working Groups work together on infrastructure-related maintenance and development efforts ([Infrastructure Chapter](https://book.the-turing-way.org/community-handbook/infrastructure)).
 - ✅ **Accessibility Monthly Meeting**s: Every second Monday of the month, at 17:00 UK time, members of the Accessibility Working Group work together on access-related chapters and guidance for the community ([Accessibility Chapter](https://book.the-turing-way.org/community-handbook/accessibility)).
+- ✅ **Book Dash Meetings**: Every month (day and time variable) between planning Book Dashes. Members of the Book Dash Working Group work together to make long term strategic decisions about our Book Dash events and they help to form and run the Book Dash planning committees [Book Dash WG repository](https://github.com/the-turing-way/bookdash).  Once the planning committee has been formed for each Book Dash, they meet more regularly (bi-weekly) in the run up to the Book Dash events in May and November each year.
+- ✅ **Community Management Meetings**: Every Tuesday, 10:00 UK time. Members of the Community Management Working Group work together to engagement and communicate with our community. Join the #community-management channel on Slack for more.
```

Comment:
I think we shouldn't write these here. It will be easy to miss this when changes are made.

It would be better to have one source of truth, which I think should be the calendar.